internal/gps/cmd.go

```diff
@@ -196,12 +198,31 @@ func runFromRepoDir(ctx context.Context, repo vcs.Repo, timeout time.Duration, c
 	return c.combinedOutput(ctx)
 }
 
+func getenvDuration(key string, def time.Duration) time.Duration {
+	seconds, err := strconv.Atoi(os.Getenv(key))
+
+	if err != nil {
+		return def
```

Comment:
Correct. It might be confusing.

I don't mind making this more user-friendly. However I will put that on hold until we have consensus that the environment variables are indeed the way to go.